### DISCUSSION TITLE
using gcc 13 with the following error, how to remedy?

### DISCUSSION THREAD

**Discussion by sprappcom** [ggml-org/llama.cpp#9069] [Q&A]
/usr/local/cuda/include/crt/host_config.h:143:2: error: #error -- unsupported GNU version! gcc versions later than 12 are not supported! The nvcc flag '-allow-unsupported-compiler' can be used to override this version check; however, using an unsupported host compiler may cause compilation failure or incorrect run time execution. Use at your own risk.
  143 | #error -- unsupported GNU version! gcc versions later than 12 are not supported! The nvcc flag '-allow-unsupported-compiler' can be used to override this version check; however, using an unsupported host compiler may cause compilation failure or incorrect run time execution. Use at your own risk.
      |  ^~~~~
make: *** [Makefile:746: ggml/src/ggml-cuda.o] Error 1

**Comment by AlanCoding** [comment]
This is also what I'm stuck on. Either of:

```
CMAKE_CUDA_FLAGS="--allow-unsupported-compiler" cmake -S . -B build -DGGML_CUDA=ON
```

or

```
CMAKE_CUDA_FLAGS="-allow-unsupported-compiler" cmake -S . -B build -DGGML_CUDA=ON
```

fails on the same error you gave, on `/usr/local/cuda/bin/../targets/x86_64-linux/include/crt/host_config.h:143:2` for me.
